rpm/percona-release.template

```diff
@@ -42,8 +42,8 @@ Source2:      RPM-GPG-KEY-Percona
 
 %post
 #
-OS=$(uname -r | cut -d '.' -f6 | awk -F'_' '{print $1}')
-if [ "x${OS}" != "xel8" ]; then
+eval $(grep VERSION_ID= /etc/os-release)
```

Comment:
it will not work on centos6

Reply:
you can try to check system-release. need to test on RH

Reply:
Works with system-release, updated

Amazon returns: `2`